backend/estimate-service/src/main/java/org/egov/works/repository/rowmapper/EstimateRowMapper.java

```diff
@@ -107,16 +107,18 @@ private void addEstimateDetails(ResultSet rs, Estimate estimate, Map<String, Est
     }
 
 
-    private JsonNode getAdditionalDetail(String columnName, ResultSet rs) {
-        JsonNode additionalDetail = null;
+    private JsonNode getAdditionalDetail(String columnName, ResultSet rs) throws SQLException {
+        JsonNode propertyAdditionalDetails = null;
```

Comment:
Why name this object as `property`AdditionalDetail? Rename it.

Reply:
done.